druid/src/widget/split.rs

```diff
@@ -44,42 +46,50 @@ impl<T> Split<T> {
             split_direction,
             min_size: 0.0,
             solid: false,
-            split_point: 0.5,
-            splitter_size: 10.0,
+            split_point_chosen: 0.5,
+            split_point_effective: 0.5,
+            splitter_size: 6.0,
+            min_splitter_area: 6.0,
             draggable: false,
             child1: WidgetPod::new(child1).boxed(),
             child2: WidgetPod::new(child2).boxed(),
         }
     }
+
     /// Create a new split panel, with a vertical splitter between two children.
     pub fn vertical(child1: impl Widget<T> + 'static, child2: impl Widget<T> + 'static) -> Self {
         Self::new(Axis::Vertical, child1, child2)
     }
+
     /// Create a new split panel, with a horizontal splitter between two children.
     pub fn horizontal(child1: impl Widget<T> + 'static, child2: impl Widget<T> + 'static) -> Self {
         Self::new(Axis::Horizontal, child1, child2)
     }
+
     /// Set container's split point as a fraction of the split dimension
     /// The value must be between 0.0 and 1.0, exclusive
     pub fn split_point(mut self, split_point: f64) -> Self {
         assert!(
             split_point > 0.0 && split_point < 1.0,
             "split_point must be between 0.0 and 1.0!"
         );
-        self.split_point = split_point;
+        self.split_point_chosen = split_point;
         self
     }
+
     /// Builder-style method to set the minimum size for both sides of the split.
     ///
     /// The value must be greater than or equal to `0.0`.
     pub fn min_size(mut self, min_size: f64) -> Self {
         assert!(min_size >= 0.0);
         self.min_size = min_size;
-
         self
     }
-    /// Set the width of the splitter bar, in pixels
-    /// The value must be positive or zero
+
+    /// Set the width of the splitter bar.
```

Comment:
```suggestion
    /// Builder-style method to set the width of the splitter bar.
```

Reply:
Should we start adding methods that allow mutation of this kind of thing at runtime (so that this distinction is meaningful)?

I think the reason we haven't thus far is the type-erased nature of children contained by built-in container widgets? But it feels like we're moving in a direction where it's acceptable to work around that or downcast out of type-erasure. So the distinction could *become* meaningful.

Reply:
I'm just trying to make this a convention in the docs for any builder-style methods, since we have decided to forgo having actual builders.

For mutating methods, my feeling is "it depends". Some cases feel really obvious (changing label color) but others I'm less clear about. I don't think we need to add them everywhere; we can see what use dictates.